lib/components/admin/mailables-window.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react'
+import { Badge } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import * as callTakerActions from '../../actions/call-taker'
+import DraggableWindow from './draggable-window'
+import Icon from '../narrative/icon'
+import {MailablesList, WindowHeader} from './styled'
+import {createLetter, MAILABLE_FIELDS} from '../../util/mailables'
+
+/**
+ * A window enabled through the Mailables module that allows Call Taker users
+ * to generate a PDF with an invoice of items to be mailed to transit customers.
+ */
+class MailablesWindow extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      mailables: []
+    }
+  }
+
+  _addMailable = (mailable) => {
+    if (!this.state.mailables.find(m => m.name === mailable.name)) {
+      const mailables = [...this.state.mailables]
+      mailables.push({...mailable, quantity: 1})
+      this.setState({mailables})
+    }
+  }
+
+  _removeMailable = (mailable) => {
+    const mailables = [...this.state.mailables]
+    const removeIndex = mailables.findIndex(m => m.name === mailable.name)
+    mailables.splice(removeIndex, 1)
+    this.setState({mailables})
+  }
+
+  _updateMailableField = (index, fieldName, value) => {
+    const mailables = [...this.state.mailables]
+    mailables[index] = {...mailables[index], [fieldName]: value}
+    this.setState({mailables})
+  }
+
+  _onClickCreateLetter = () => createLetter(this.state, this.props.mailablesConfig)
+
+  _updateField = (evt) => {
```

Comment:
This had me a little confused at first. I think this could be better described as a letter field, so maybe it'd make sense to rename this to `_updateLetterField`?